start_cerberus.py

```diff
@@ -69,27 +69,16 @@ def main(cfg):
 
         # Create slack WebCleint when slack intergation has been enabled
         if slack_integration:
-            try:
-                slackcli.initialize_slack_client()
-            except Exception as e:
-                slack_integration = False
-                logging.error("Couldn't create slack WebClient. Check if slack env "
-                              "varaibles are set. Exception: %s" % (e))
-                logging.info("Slack integration has been disabled.")
-
-        # Remove 'inspect_data' directory if it exists.
-        # 'inspect_data' directory is used to collect
-        # logs, events and metrics of the failed component
-        if os.path.isdir("inspect_data/"):
-            logging.info("Deleting existing inspect_data directory")
-            runcommand.invoke("rm -R inspect_data")
+            slack_integration = slackcli.initialize_slack_client()
+
+        if inspect_components:
```

Comment:
Maybe move the delete operation to line 144 where the data collection happens when inspect_components is set to true? Thoughts? Also, let's add a log message when inspection is enabled.

Reply:
Delete operation needs to be outside the while loop as the directory is deleted only once when Cerberus starts monitoring.

Reply:
Makes sense. Thanks.